scripts/bench/topology/topology0.yaml

```diff
@@ -2,18 +2,24 @@ nodes:
   node0:
     type: core
     region: undefined
-    static-routes: [[node1], [node2]]
+    static-routes: [[node1], [node2], [node4]]
     addr: 127.0.0.1
     port: 3000
   node1:
     type: core
     region: undefined
-    static-routes: [[node0], [node2]]
+    static-routes: [[node0], [node2], [node4]]
     addr: 127.0.0.1
     port: 3001
   node2:
     type: core
     region: undefined
-    static-routes: [[node0], [node1]]
+    static-routes: [[node0], [node1], [node4]]
     addr: 127.0.0.1
     port: 3002
+  node4:
```

Comment:
node3 is an edge node that will subscribe to the relay, so it's not mentioned in the topology file.